Library/Formula/readline.rb

```diff
@@ -33,7 +33,7 @@ class Readline < Formula
 
   def install
     ENV.universal_binary
-    system "./configure", "--prefix=#{prefix}", "--enable-multibyte"
+    system "./configure", "--prefix=#{prefix}", "--enable-multibyte", ("--with-curses" if OS.linux?)
     system "make", "install"
```

Comment:
Does this patch fix an error message that you observed? If so, can you report the error message?
Please submit this `readline` chunk in a separate PR.